system/validating-admission-policy/templates/protect-crds.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: protect-crds.cloud.sap
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["*"]
+      apiVersions: ["*"]
+      operations:  ["DELETE"]
```

Comment:
`"DELETECOLLECTION"` is missing.

Reply:
Ok, so `"DELETECOLLECTION"` [does not exist](https://github.com/kubernetes/kubernetes/blob/1bcfd5cee7905ee7c2b17cec0ecd88609693f44c/pkg/apis/admissionregistration/types.go#L1068) on `ValidatingAdmissionPolicies`, but it exists for the `verbs` field on `(Cluster)Roles`. 🙄